index.js

```diff
@@ -20,6 +20,7 @@ class VPCPlugin {
    */
   updateVpcConfig() {
     const awsCreds = this.serverless.providers.aws.getCredentials();
+    awsCreds.region = this.serverless.providers.aws.getRegion();
```

Comment:
alright if i understand this correctly, your updating this guy: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html

Reply:
i couldn't figure out why you were updating the creds with region then i realized you were just re-using the obj.
how about having 2 objects, one for creds and one for region then calling update() twice?
of having a generic object, and updating it with creds and region then using that to call update()?

bit more verbose and more clear

Reply:
ahh damn... you merged before i could write my comment :/

Reply:
no worries, it was a nitpick